bin/board.rb

```diff
@@ -0,0 +1,15 @@
+class Board
+  attr_accessor :board, :counter
```

Comment:
- [ ] [REQUIRED] Only uses attr_accessors if you really need it since doing this means that property can be written to from outside the class.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))